### DISCUSSION TITLE
How can I change the color for correct answers?

### DISCUSSION THREAD

**Discussion by DBShorts** [Epicpkmn11/WordleDS#8] [Support and Suggestions]
I am working on a port of Canuckle. The correct answers in Canuckle show red instead of green. Is there a way to change the green to red?

**Comment by Epicpkmn11** [comment]
You should just need to edit the colors at [gfx.cpp:24](https://github.com/Epicpkmn11/WordleDS/blob/main/source/gfx.cpp#L24) (the top screen and keyboard) and [gfx.cpp:41](https://github.com/Epicpkmn11/WordleDS/blob/main/source/gfx.cpp#L41) (the text in the statistics menu) (and also the high contrast ones right below those if you want to change that too), they're in BGR15 so use [this](https://xn--rck9c.xn--tckwe/en/color.html) or [this](https://www.conradshome.com/html2bgr15/) to convert normal colors.

Probably also want to edit [main.cpp:75](https://github.com/Epicpkmn11/WordleDS/blob/main/source/main.cpp#L75) to change the :green_square: to :red_square:.

**Comment by DBShorts** [comment]
Thank you so much, this is very helpful! I already changed the emojis, i was just stuck on the actual on screen letters.

**Comment by DBShorts** [comment]
Another question though, why is there such a long row of colors at [gfx.cpp:24](https://github.com/DBShorts/CanuckleDS/blob/patch-1/source/gfx.cpp#L24)? Is it to make some sort of fading in animation?

**Comment by Epicpkmn11** [comment]
First one is transparent, then the next two are the border and background, then the next 13 are just a gradient for smoother antialiasing on the letters.
![スクリーンショット 2022-03-20 13 43 52](https://user-images.githubusercontent.com/41608708/159177609-9038e9f5-6c11-4508-9766-48dc9c5e210c.png)
(the Foreground Palettes)

**Comment by Epicpkmn11** [comment]
Transferring to a discussion as this isn't an issue in Wordle DS, if you have more questions feel free to ask in the discussion.
